libr/parse/parse.c

```diff
@@ -328,9 +328,15 @@ static int filter(RParse *p, RFlag *f, char *data, char *str, int len, bool big_
 		}
 		if (p->hint) {
 			int pnumleft, immbase = p->hint->immbase;
-			char num[256], *pnum, *tmp;
+			char num[256] = {0}, *pnum, *tmp;
 			bool is_hex = false;
 			int tmp_count;
+			if (p->hint->offset) {
+				snprintf (num, sizeof (num), "%s", p->hint->offset);
```

Comment:
Why, just pass this to the expr below. Also offset is a string? And.. if so, this is just an strcpy